src/Microsoft.Health.Dicom.Api/Controllers/ExtendedQueryTagController.cs

```diff
@@ -48,7 +48,7 @@ public ExtendedQueryTagController(IMediator mediator, ILogger<ExtendedQueryTagCo
         [HttpPost]
         [Route(KnownRoutes.ExtendedQueryTagRoute)]
         [AuditEventType(AuditEventSubType.AddExtendedQueryTag)]
-        public async Task<IActionResult> PostAsync([FromBody] IEnumerable<ExtendedQueryTagEntry> extendedQueryTags)
+        public async Task<IActionResult> PostAsync([FromBody] IEnumerable<AddExtendedQueryTagEntry> extendedQueryTags)
         {
             _logger.LogInformation("DICOM Web Add Extended Query Tag request received, with extendedQueryTags {extendedQueryTags}.", extendedQueryTags);
 
```

Comment:
Call StatusCode without respose?
 ```
return StatusCode(
               (int)HttpStatusCode.Accepted, response);
```

=>
```
 return StatusCode(
               (int)HttpStatusCode.Accepted);
```

Reply:
I'd like to keep an empty response for now since we will be adding properties to it soon. Consumers shouldn't face too much of an issue with an empty response body for now.